### ISSUE TITLE
Incompatible with PyTest

### ISSUE THREAD

**Issue by kakyoism** [ebu/ebu_adm_renderer#50]
First of all, thanks for this little tool! It proves very useful in our projects.

However, we now have a dependency conflict problem, see the error report from Poetry below 

```sh
  SolverProblemError

  Because no versions of ear match <1.0.0 || >1.0.0,<1.0.1 || >1.0.1,<1.1.0 || >1.1.0,<1.1.1 || >1.1.1,<1.1.2 || >1.1.2,<1.2.0 || >1.2.0,<2.0.0 || >2.0.0
   and ear (1.0.0) depends on attrs (>=17.4,<18.0), ear (<1.0.1 || >1.0.1,<1.1.0 || >1.1.0,<1.1.1 || >1.1.1,<1.1.2 || >1.1.2,<1.2.0 || >1.2.0,<2.0.0 || >2.0.0) requires attrs (>=17.4,<18.0).
  And because ear (1.0.1) depends on attrs (>=17.4,<18.0)
   and ear (1.1.0) depends on attrs (>=17.4,<18.0), ear (<1.1.1 || >1.1.1,<1.1.2 || >1.1.2,<1.2.0 || >1.2.0,<2.0.0 || >2.0.0) requires attrs (>=17.4,<18.0).
  And because ear (1.1.1) depends on attrs (>=17.4,<18.0)
   and ear (1.1.2) depends on attrs (>=17.4,<18.0), ear (<1.2.0 || >1.2.0,<2.0.0 || >2.0.0) requires attrs (>=17.4,<18.0).
  And because ear (1.2.0) depends on attrs (>=17.4,<18.0)
   and ear (2.0.0) depends on attrs (>=17.4,<18.0), every version of ear requires attrs (>=17.4,<18.0).
  And because wavinfo (1.6.3) depends on ear (*)
   and pytest (6.2.0) depends on attrs (>=19.2.0), wavinfo (1.6.3) is incompatible with pytest (6.2.0).
  So, because MyProj depends on both pytest (6.2.0) and wavinfo (1.6.3), version solving failed.

```

Can we do something here?

Thanks again!

**Comment by tomjnixon** [comment]
Hi,

I'm pretty sure this is fixed in master -- i don't use poetry, but modern pip versions should have the same behaviour.

I really need to make a new release...

**Comment by tomjnixon** [comment]
Just released 2.1.0 which should solve this; please re-open this if it doesn't.
